CHANGELOG.md

```diff
@@ -2,6 +2,90 @@
 
 About the past changelog entries, see [old CHANGELOG](CHANGELOG-v4.md) instead.
 
+## Release v5.0.2 - 2023/11/29
+
+### News
+
+* Update fluentd to 1.16.3
+* Update bundled gems
+* msi: support path which contains space or parenthesis (#589)
+* deb: fixed system user/group name in logrotate config (#592,#594)
+  * It fixes a bug that unknown user error was reported.
+* rpm: fixed to create fluentd user as system account (#596)
+  * It fixes a bug that /var/lib/fluent directory was created unexpectedly.
+* rpm: changed to keep system account after removing fluent-package. (#598)
+  * In the previous versions, there was a bug that group was not cleanly removed
+    when the package was upgraded from td-agent v4.
+    This change makes reinstall/downgrade friendly.
+* Update fluent-diagtool to v1.0.3
+  * It supports fluent-package!
```

Comment:
```suggestion
  * It supports fluent-package! (On Windows, all feature isn't available yet) 
```

Reply:
And drop bellow sentence.

Reply:
What is its aim?
I'd like to consider whether there is another expression because it sounds to me as if Windows was supported on td-agent.

Reply:
>  for simplicity

Sorry, I missed this part.

Reply:
I think "fluent-package support" and "Windows support" are parallel, so if we should reduce the number of lines, how about this?

> It supports fluent-package and Windows! (On Windows, all features are not available yet. )

And drop the below sentence.

Reply:
I felt the discomfort the following part. It was completely right, but...

```
* It supports ...
* It supports ...
* It supports ...
```

Reply:
Oh, I see...
Is it because the same expressions are repeated? Hmm...

Reply:
> Is it because the same expressions are repeated?

Hmm, but I think it is usual for CHANGELOG... Hmm...

Reply:
Maybe we can omit `it`.

* Supports ...
* Supports ...
* ...

Reply:
I have tried to make it a CHANGELOG-like bulleted list.
If you still feel strange, please let me know.

Reply:
> Is it because the same expressions are repeated? Hmm...

Yes. but I have no idea more better rephrase. :thinking: